assets/app/view/company.rb

```diff
@@ -75,8 +76,10 @@ def render
         },
         on: { click: onclick },
       }
-
-      props[:style]['background-color'] = 'lightblue' if selected?
+      if selected?
```

Comment:
we could make selected a class, you could add it to index.rb

then do
h("div.selected_entity") to add the class or something

Reply:
Well, if you’d like to use more CSS/SASS I wouldn’t mind. I took your answers yesterday as "avoid CSS whenever possible" ;)

Reply:
haha, no i mean you can use it when it's necessary or beneficial

Reply:
Ok, got it. It’s actually ok in this case.
There are some other instances where it’s actually easier, less errorprone and more maintenance-friendly to use CSS. All those "color: black"s e. g. and other redundant styles, but I didn’t want to push "my agenda" with the first PR ;-)

Reply:
i wouldn't see it that way unless you rewrote the whole thing in css.

if it makes the code cleaner and less redundant to use css, by all means do it.
for example i have a css class called margined or button etc

Reply:
At least it makes for cleaner and more legible HTML. CSS tends to get very convoluted for larger projects, so it’s a slippery slope for sure. So I do get why you prefer another way, even when it’s not dynamically changed.
But having more CSS in one place might motivate someone without any ruby knowledge to improve the design/presentation of the site. (Back in April I would’ve put in a PR instead of an issue – if there had been a simple .css.)
I might tackle CSS after this PR and tile/map-themes are done.